doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1020 @@
+            )
+        return result_array
+
+    def __check_emergency(self, data: Bool):
+        """If an emergency stop is needed first check if we are
+        in parking behavior. If we are ignore the emergency stop.
+
+        Args:
+            data (Bool): True if emergency stop detected by collision check
+        """
+        # self.loginfo("Emergency stop detected")
+        if not self.__curr_behavior == bs.parking.name:
```

Ruff (0.8.2)

Use `self.__curr_behavior != bs.parking.name` instead of `not self.__curr_behavior == bs.parking.name`

Replace with `!=` operator

(SIM201)